### ISSUE TITLE
Build with JDK 9 fails

### ISSUE THREAD

**Issue by mchalupa** [sosy-lab/llvm-j#7]
Hi,

when I try to build the master branch I'm getting this error:

```
    [mkdir] Created dir: /home/mchalupa/src/llvm-j/.apt-generated
    [javac] Compiling 1 source file to /home/mchalupa/src/llvm-j/bin
    [javac] Compiling 18 source files to /home/mchalupa/src/llvm-j/bin
    [javac] /home/mchalupa/src/llvm-j/src/main/java/org/sosy_lab/llvm_j/Module.java:35: error: package com.sun.istack.internal does not exist
    [javac] import com.sun.istack.internal.Nullable;
    [javac]                               ^
    [javac] /home/mchalupa/src/llvm-j/src/main/java/org/sosy_lab/llvm_j/Utils.java:32: error: package com.sun.istack.internal does not exist
    [javac] import com.sun.istack.internal.Nullable;
    [javac]                               ^
    [javac] /home/mchalupa/src/llvm-j/src/main/java/org/sosy_lab/llvm_j/Module.java:187: error: cannot find symbol
    [javac]   public @Nullable String getOriginFileName() {
    [javac]           ^
    [javac]   symbol:   class Nullable
    [javac]   location: class Module
    [javac] /home/mchalupa/src/llvm-j/src/main/java/org/sosy_lab/llvm_j/Utils.java:45: error: cannot find symbol
    [javac]   static boolean llvmBoolToJavaBool(@Nullable LLVMLibrary.LLVMBool pBool) {
    [javac]                                      ^
    [javac]   symbol:   class Nullable
    [javac]   location: class Utils
    [javac] 4 errors
```

I guess I may have a different version of JDK?

```
openjdk version "9"
OpenJDK Runtime Environment (build 9+181)
OpenJDK 64-Bit Server VM (build 9+181, mixed mode)
```

**Comment by mdangl** [comment]
Yes, as far as I know, com.sun.* is not part of the java standard library and should therefore *never* be used in application code. This needs to be fixed.

**Comment by mchalupa** [comment]
Should be fixed by #9
